doc/env_vars.md

```diff
@@ -194,6 +194,12 @@ These config vars enable and configure browser security response headers from th
 | `LRSQL_SEC_HEAD_CROSS_DOMAIN` | `secHeadCrossDomain` | `X-Permitted-Cross-Domain-Policies` | `none`                                |
 | `LRSQL_SEC_HEAD_CONTENT`      | `secHeadContent`     | `Content-Security-Policy`           | `object-src 'none'; script-src 'unsafe-inline' 'unsafe-eval' 'strict-dynamic' https: http:;` |
 
+#### Proxy Settings
```

Comment:
there actually already _are_ some proxy settings under webserver / Hosts&Ports. So either move them down here or move this up there.

Also you should give a better description. Something like:

"Allows a call to the xAPI endpoints to use the admin account credentials / authorization instead of a call with raw xAPI credentials. Necessary when the xAPI endpoint is behind additional security/SSO and cannot have custom credentials in Authorization header."

Reply:
changed accordingly

Reply:
@invaliduser Make sure that you remove this section entirely

Reply:
removed, ty